man/nvim.1

```diff
@@ -337,10 +337,24 @@ Print version information and exit.
 .Sh ENVIRONMENT
 .Bl -tag -width Fl
 .It Ev VIM
-Used to locate various user files, such as the user's nvimrc.
+Used to locate various user files, such as the user's init.vim.
 .It Ev VIMRUNTIME
 Used to locate run time files, such as on-line documentation and
 syntax highlighting definitions.
+.It Ev XDG_CONFIG_HOME
+Path to use for the user-local runtime directory, see
```

Comment:
There actually is an `$XDG_RUNTIME_DIR`, so consider changing this to:

    Path to user-local configuration, see